### DISCUSSION TITLE
make an AUR package

### DISCUSSION THREAD

**Discussion by ghost** [rafaelmardojai/firefox-gnome-theme#242] [Q&A] [answered]

**Comment by ispanos** [comment]
Is that even possible?

**Comment by rafaelmardojai** [comment]
You can do the same as the Manjaro community fork.

https://gitlab.manjaro.org/packages/community/themes/firefox-gnome-theme-maia
https://gitlab.manjaro.org/Chrysostomus/firefox-gnome-theme-maia

But personally I don't have time nor interest.

**Comment by emansom** [comment, marked as answer]
AUR package [`firefox-gnome-theme-git`](https://aur.archlinux.org/packages/firefox-gnome-theme-git) has been created by @joickle.
